docs/sidebars.js

```diff
@@ -1,31 +1,148 @@
-/**
- * Creating a sidebar enables you to:
- - create an ordered group of docs
- - render a sidebar for each doc of that group
- - provide next/previous navigation
-
- The sidebars can be generated from the filesystem, or explicitly defined here.
-
- Create as many sidebars as you want.
- */
-
-// @ts-check
-
-/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
-const sidebars = {
-  // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{type: 'autogenerated', dirName: '.'}],
-
-  // But you can create a sidebar manually
-  /*
-  tutorialSidebar: [
-    {
-      type: 'category',
-      label: 'Tutorial',
-      items: ['hello'],
+module.exports = {
+  docs: [
+    'Getting-Started/welcome-to-langflow',
+    {
+      type: 'category',
+      label: 'Getting Started',
```

Comment:
Per the Google style guide, avoid gerunds in titles. Use "Get started" instead.

Reply:
Follow on here #4813